.github/workflows/publish-npm.yml

```diff
@@ -0,0 +1,19 @@
+name: "NPM Publish"
+on:
+  release:
+    types: [published]
+#  pull_request:
+
+jobs:
+  NPM-Publish:
+    permissions: # Must change the job token permissions to use JWT auth
+      contents: read
+      id-token: write
+
+    uses: celo-org/reusable-workflows/.github/workflows/npm-publish.yaml@v1.11.1
+    with:
+      node-version: 16
+      package-dir: "packages/snap"
+      akeyless-api-gateway: https://api.gateway.akeyless.celo-networks-dev.org
+      akeyless-github-access-id: p-kf9vjzruht6l
+      akeyless-token-path: /static-secrets/apps-tooling-circle/npm-publish-token
```

Comment:
FYI ..i got these from @pputman-clabs 
Who also help setup this repo for permissions 👍